.github/workflows/ci.yml

```diff
@@ -36,18 +35,17 @@ jobs:
         uses: actions/cache@v1
         id: libtorch-cache
         with:
-          path: libtorch
-          key: libtorch.zip
-      - name: Download and install libtorch
-        if: ${{ matrix.working_directory == 'torchx' && steps.libtorch-cache.outputs.cache-hit != 'true' }}
-        run: |
-          wget https://download.pytorch.org/libtorch/lts/1.8/cpu/libtorch-shared-with-deps-1.8.2%2Bcpu.zip -q -O libtorch.zip
-          unzip -qq libtorch.zip
+          path: ./torchx/cache/libtorch
+          key: ./torchx/cache/libtorch.zip
```

Comment:
@jonatanklosko the cache action isn't reacting to `matrix.working-directory` properly! Now I got a cache hit :)

Reply:
(and we didn't re-download libtorch because of it)

Reply:
Oh, turns out `defaults.run` doesn't apply to actions (`uses`). So we need to update the mix cache action too by manually building path relative to matrix.working_directory